### PR TITLE
Fix to PG screenshot and process

### PR DESCRIPTION
**Pull request by Vernholio**
* Updated screenshot.
* Updated steps to create a new Placement Group, based new screenshot.

### CODE REVIEW

**Review by kutlay — APPROVED** [review]
The new screenshot and the instructions look great. Thanks!

File: docs/products/compute/compute-instances/guides/placement-groups/index.md [hunk 1]
```diff
@@ -79,22 +79,22 @@ Make sure you understand how placement groups work. Have a look at [Affinity, en
 - Once you create your placement group, you *can't change* its Affinity Type Enforcement.
```
[the remark this answers is not in the document]
Comment (Vernholio) [reply docs/products/compute/compute-instances/guides/placement-groups/index.md:79]: I know this contradicts traditional markdown numbered lists. Matt's told me that they use "lazy numbering," meaning just set everything to "1."  I'm guessing the Hugo publishing mechanism addresses this somehow.

[the remark this answers is not in the document]
Comment (crystallearobertson) [reply docs/products/compute/compute-instances/guides/placement-groups/index.md:79]: Ah, ok. I use lazy numbering too. Weird that Netlify doesn't render it correctly, but if it renders ok when you run Hugo Server, all good. :)

File: docs/products/compute/compute-instances/guides/placement-groups/index.md [hunk 2]
```diff
@@ -79,22 +79,22 @@ Make sure you understand how placement groups work. Have a look at [Affinity, en
...
 {{< /note >}}
 
-3.  When you're ready, click **Create Placement Group**. A summary of your group is revealed.
+1. When you're ready, click **Create Placement Group**.
 
-4.  Select the **Linodes (0)** tab.
+2. Click the Label for you new placement group. A summary of your group is shown.
 
-1.  Click **Assign Linode to Placement Group**. The Assign Linodes form opens.
+3. Click **Assign Linode to Placement Group**. The Assign Linodes to \<Label\> drawer opens.
```
[the remark this answers is not in the document]
Comment (Vernholio) [reply docs/products/compute/compute-instances/guides/placement-groups/index.md:86]: The "Linodes in <region>" field is clickable, while the "Assign Linodes to <Label>" drawer isn't. That was my logic there.

**Review by crystallearobertson — APPROVED** [review]

**Pull request merged**